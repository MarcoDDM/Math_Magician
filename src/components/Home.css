.home {
  width: 50%;
  border: 1px solid;
  position: absolute;
  top: 7rem;
}
