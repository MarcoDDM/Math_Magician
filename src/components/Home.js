import React from 'react';
import './Home.css';

export const Home = () => (
  <div className="home">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Proin faucibus, elit eget accumsan congue, ante elit iaculis dolor,
    eget luctus lorem odio eu leo. Ut eget lacinia mauris.
    Donec a lectus non diam scelerisque iaculis at ac massa.
    Nullam laoreet risus odio, vitae euismod ex commodo vel.
    Suspendisse potenti.
    Vivamus quis leo eu massa tincidunt semper at ut metus.
    Nunc finibus, tortor eu pharetra efficitur, tortor odio bibendum tellus,
    vitae faucibus est eros vel ipsum.

  </div>

);

export default Home;
