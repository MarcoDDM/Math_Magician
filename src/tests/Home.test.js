import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/Home';

describe('Test suite for the Home component', () => {
  let home;
  beforeEach(() => {
    render(<Home />);
    home = screen.getByTestId('home');
  });

  it('Should test if Home component is inside the document', () => {
    expect(home).toBeInTheDocument();
  });

  it('Should render a snapshot of the Home component', () => {
    expect(home).toMatchSnapshot();
  });
});
