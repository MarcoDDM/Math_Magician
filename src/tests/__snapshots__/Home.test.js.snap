// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for the Home component Should render a snapshot of the Home component 1`] = `
<div
  class="home"
  data-testid="home"
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin faucibus, elit eget accumsan congue, ante elit iaculis dolor, eget luctus lorem odio eu leo. Ut eget lacinia mauris. Donec a lectus non diam scelerisque iaculis at ac massa. Nullam laoreet risus odio, vitae euismod ex commodo vel. Suspendisse potenti. Vivamus quis leo eu massa tincidunt semper at ut metus. Nunc finibus, tortor eu pharetra efficitur, tortor odio bibendum tellus, vitae faucibus est eros vel ipsum.
</div>
`;
